### PR TITLE
homepage open menu tab

### PR DESCRIPTION
**Pull request by JeH9o**
Home_screen edit: Changed back icon to menu icon.
Users can open the menu icon when click on it. Menu build as side sheet.
The menu also can be closed.

### CODE REVIEW

File: lib/screens/home_screen.dart
Comment (elviazamora338) [inline lib/screens/home_screen.dart:1]: Go ahead an incorporate a test for when the user click menu icon they should see the menu options

File: test_driver/app_test.dart
Comment (elviazamora338) [inline test_driver/app_test.dart:1]: Go ahead an incorporate a test for when the user clicks menu icon they should see the menu options

**Comment by elviazamora338** [comment]
Make sure to include description of what you added or changed in the code please in your next PR to make it easier to review for others

**Comment by JeH9o** [comment]
Home_screen edit: Changed back icon to menu icon. 
Users can open the menu icon when click on it. Menu build as side sheet.
The menu also can be closed.

**Review by elviazamora338 — APPROVED** [review]
LGTM

**Pull request merged**